src/main/java/uk/gov/ida/verifylocalmatchingserviceexample/dao/VerifiedPidDAO.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.ida.verifylocalmatchingserviceexample.dao;
+
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+public interface VerifiedPidDAO {
```

Comment:
Can we have an easier to read name for this? Maybe VerifiedPid would do?

Reply:
Fair point. Will change it.